content/client-lib-development-guide/features.textile

```diff
@@ -492,6 +491,7 @@ h3(#realtime-channels). Channels
 *** @(RTS3c1)@ If a new set of @ChannelOptions@ is supplied to @Channels#get@ that would trigger a reattachment of the channel if supplied to @Channel#setOptions@ per "@RTL16a@":#RTL16a, it must raise an error, informing the user that they must use @Channel#setOptions@ instead
 * @(RTS4)@ @Channels#release@ function:
 ** @(RTS4a)@ Detaches the channel and then releases the channel resource i.e. it's deleted and can then be garbage collected
+** @(RTS4b)@ Calling @release()@ with a channel name that corresponds to a channel that is in any state other than @INITIALIZED@, @DETACHED@, or @FAILED@ must, instead of releasing the channel, raise an error with code @90001@
```

Comment:
So now these two spec items contradict each other - if you call release on an attached channel, should it detach then release, or raise an error?

My suggestion in the slack thread was just get rid of RSN4c entirely

Reply:
@SimonWoolf I lost context after a long gap. Removing `RTS4b` is what we are looking for, is it?

Reply:
yes

Reply:
@SimonWoolf removed

Reply:
Resolving this conversation on @SimonWoolf's behalf.